### PR TITLE
Default enhanced colors, remove bold statics, and remove experiment

### PR DESCRIPTION
**Pull request by JoeRobich**
We will leave the preview features checkbox in for users who wish to opt-out.

<details><summary>Ask Mode template</summary>

### Customer scenario

When using C# or VB we will set new default colors for Methods, Operator Overloads, Control Keywords, Local variables and Parameters. 

### Bugs this fixes

None, this is just defaulting a feature which we shipped in Preview 2.

### Workarounds, if any

None

### Risk

Low.

### Performance impact

none

### Is this a regression from a previous update?

No

### Root cause analysis

NA

### CODE REVIEW

File: src/VisualStudio/Core/Def/Implementation/Experimentation/EnhancedColorExperiment.cs [hunk 1]
```diff
@@ -246,7 +227,7 @@ public void TrySetEnhancedColors(Guid themeId)
                     UpdateColorItem(colorItemMap, ClassificationTypeNames.OperatorOverloaded, DarkThemeMethodYellow);
                     UpdateColorItem(colorItemMap, ClassificationTypeNames.ControlKeyword, DarkThemeControlKeywordPurple);
                     UpdateColorItem(colorItemMap, ClassificationTypeNames.StructName, DarkThemeStructMint);
-                    UpdateColorItem(colorItemMap, ClassificationTypeNames.StaticSymbol, DefaultForegroundColor, DefaultBackgroundColor, isBold: true);
+                    UpdateColorItem(colorItemMap, ClassificationTypeNames.StaticSymbol, DefaultForegroundColor, DefaultBackgroundColor);
```
Comment (sharwell) [inline src/VisualStudio/Core/Def/Implementation/Experimentation/EnhancedColorExperiment.cs]: ❓ Why did the `isBold` value change here?

Comment (JoeRobich) [reply src/VisualStudio/Core/Def/Implementation/Experimentation/EnhancedColorExperiment.cs]: We are looking for feedback that that bold statics are a desired default.

Comment (mavasani) [reply src/VisualStudio/Core/Def/Implementation/Experimentation/EnhancedColorExperiment.cs]: +1 for this change from me, I hate bold statics :-).

**Comment by CyrusNajmabadi** [comment]
Bold statics are the devil.

**Pull request merged**